src/components/Drawer.jsx

```diff
@@ -147,6 +147,8 @@ const ChurchDrawer = React.memo(function ChurchDrawer({ handleDrawerClose, drawe
       variant="persistent"
       anchor="left"
       open={open}
+      onClose={handleDrawerClose}
+      onOpen={() => {}}
```

Comment:
Why we are sending onOpen like this ?

Reply:
Actually ,it was not my task , onOpen is required props  Component, I added for  remove   console.log warning